casper/contracts/simple_casper.v.py

```diff
@@ -581,24 +592,30 @@ def slash(vote_msg_1: bytes <= 1024, vote_msg_2: bytes <= 1024):
         slashing_condition_detected = True
     assert slashing_condition_detected
 
-    # Delete the offending validator, and give a 4% "finder's fee"
+    # Slash the offending validator, and give a 4% "finder's fee"
     validator_deposit: int128(wei) = self.deposit_size(validator_index_1)
     slashing_bounty: int128(wei) = floor(validator_deposit / 25)
     deposit_destroyed: int128(wei) = validator_deposit - slashing_bounty
-    self.total_destroyed += deposit_destroyed
+    self.total_slashed[self.current_epoch] += validator_deposit  # should this be current_epoch + 1?
```

Comment:
`self.current_epoch` is the right one to me.
if a 100 is slashed at epoch 5, so total_slashed[5] = 100. and then epoch 6 is initialized, where `self.total_slashed[epoch] = self.total_slashed[epoch - 1]` get you total_slashed[6] == total_slashed[5] == 100. 
if we are using `current_epoch + 1` the 100 will be erased, we'll have total_slashed[5] = 0 and total_slashed[6] == total_slashed[5] == 0.

Reply:
you're right! thanks